### PR TITLE
fix: Virtual Events: Enhance "Join Event" functionality and side panel style

### LINKED ISSUE #6322: Virtual Events: Enhance "Join Event" functionality and side panel style
We can further improve the online event experience according to some user feedback. Please make the following changes:

* Make side panel event name into a link and take out external link icon
* Move right-side bar "Join Event" button above date on the right side bar to the top position
* Make "Join Event" button on right side bar full width of the side bar (similar to tweet area), make it into blue color to be more visible and increase the font size
* Take out "Join Video" button from right-side bar. It is sufficient to have it on the bottom and we have the "Join Event" button now here.
* If the attendee already participates in the event (meaning they already have the side-panel as part of their UI), but the panel is minimized the button "Join Event" should always expand the side panel

Screenshot from 2021-01-11 22-52-49

### PR DESCRIPTION
**Pull request by MeghalBisht**
Fixes #6322

#### Short description of what this resolves:

- [x] Make side panel event name into a link and take out external link icon
- [x] Move right-side bar "Join Event" button above date on the right side bar to the top position
- [x] Make "Join Event" button on right side bar full width of the side bar (similar to tweet area), make it into blue color to be more visible and increase the font size
- [x] Take out "Join Video" button from right-side bar. It is sufficient to have it on the bottom and we have the "Join Event" button now here.
- [ ] If the attendee already participates in the event (meaning they already have the side-panel as part of their UI), but the panel is minimized the button "Join Event" should always expand the side panel

zo

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by MeghalBisht** [comment]
@iamareebjamal  For the last point of the issue - **If the attendee already participates in the event (meaning they already have the side-panel as part of their UI), but the panel is minimized the button "Join Event" should always expand the side panel**

The **shown** property should be controlled from join-event component as well, so I tried lifting it to the public.js controller and then passed it to both join-video and side-panel components, but it doesn't seem to work, any inputs?

**Comment by iamareebjamal** [comment]
I'm confused, let's discuss this in the meeting

**Comment by MeghalBisht** [comment]
@iamareebjamal  Please share the details for the same

**Comment by iamareebjamal** [comment]
Next one is on Tuesday 7 PM. Join the Gitter channel, I always mention it there

**Comment by mariobehling** [comment]
This looks already pretty good. Hope you can finalize the opening of the side panel.

Also, please make the spacing between "widget areas" the same. The spacing below the "Join Event" button should be same as between the following areas (date/time and add calendar, add to calendar and icons, icons and tweet etc.).

Screenshot from 2021-01-20 01-19-35

File: app/components/public/stream/join-video.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ import Loader from 'open-event-frontend/services/loader';
 
 interface Args {
   videoStream: VideoStream;
-  event: Event
+  event: Event,
```
Comment (iamareebjamal) [inline app/components/public/stream/join-video.ts:11]:
```suggestion
  event: Event
```

**Review by mariobehling — DISMISSED** [review]
Working for me. Last point could be solved in a separate PR.

**Pull request merged**